beacon-chain/blockchain/execution_engine.go

```diff
@@ -183,7 +184,7 @@ func (s *Service) getPayloadHash(ctx context.Context, root []byte) ([32]byte, er
 // notifyForkchoiceUpdate signals execution engine on a new payload.
 // It returns true if the EL has returned VALID for the block
 func (s *Service) notifyNewPayload(ctx context.Context, postStateVersion int,
-	postStateHeader *enginev1.ExecutionPayloadHeader, blk interfaces.SignedBeaconBlock) (bool, error) {
+	postStateHeader nativetypes.ExecutionPayloadHeader, blk interfaces.SignedBeaconBlock) (bool, error) {
```

Comment:
This `ExecutionPayloadHeader` interface should be moved into `consensus-types/interfaces/beacon_block.go`